### DISCUSSION TITLE
Fluctuating training and validation accuracies

### DISCUSSION THREAD

**Discussion by Mousa-saleh** [neuropoly/axondeepseg#556] [Bug reports, suggestions for code refactoring]
Hello all,

I trained the model with 6 images (1360*1024) for training and two for validation. I got high fluctuation in the accuracies(attached).

I used  learning rate =1e-4
decay steps= 100 
decay rate =0.95
dropout= 0
Please help me figure out the problem!

Thanks,
![image](https://user-images.githubusercontent.com/85881282/129280178-4d0701cb-28a1-4878-9e24-0ffb2aa70c68.png)

**Comment by hermancollin** [comment]
Hi @Mousa-saleh
1. Would it be possible to have a look at the train/validation images? I would be curious to see the level of class imbalance of your samples.
2. Can you provide the exact config you used for training?

Also, what exactly do you mean by fluctuation? Do you mean the "oscillations" present for both training and validation curves? 

Intuitively, my guess would be that the learning rate is too high.

**Reply by Mousa-saleh** [reply]
Sorry for the late reply, I had to buy a new laptop.
I am facing a GPU problem. 
I think I should create a new thread.

Thanks,

**Reply by jcohenadad** [reply]
Yes, a new thread would be useful. Can you please cross-reference your new thread here so it is easier for us to follow where the conversation is going. Thank you.

**Reply by Mousa-saleh** [reply]
this is the new thread #560

Thanks,
